dist/t/spec/spec_helper.rb

```diff
@@ -19,6 +18,7 @@
   end
   # Limits the available syntax to the non-monkey patched
   config.disable_monkey_patching!
+  config.include Capybara::DSL
```

Comment:
But then you should c&p right ;)
``
config.include Capybara::DSL, :type => :feature
``